taskcluster/app_services_taskgraph/target_tasks.py

```diff
@@ -10,18 +10,19 @@
 def target_tasks_pr_skip(full_task_graph, parameters, graph_config):
     return []
 
-@_target_task('preview')
-def target_tasks_preview(full_task_graph, parameters, graph_config):
-    # Run all tasks for preview builds
+@_target_task('release')
+def target_tasks_release(full_task_graph, parameters, graph_config):
+    # Run all tasks for release builds
```

Comment:
What's the intent of this target task phase exactly? Shouldn't it be called `full` or something?

Fyi I'll be adding new methods for the `release_promotion` aspects, so I'm a bit unclear if this one will still be needed or not.

Reply:
The intention is to run the tasks needed for a release (nightly or otherwise).  `full` is probably a better name, but we also have `pr-full` which is run when you add `[ci: full]` in the PR title.  It's all a bit confusing to me.

Maybe `release` and `pr-full` should both be renamed to `full` and the only difference between a release, nightly, and a PR with `[ci: full]` is the indexing.

Reply:
Yeah that latter suggestion sounds sensible, though we don't need to worry about it now. I was mostly curious.

For release promotion, we'll schedule only the tasks that are needed for the actual release. Things like builds and tests presumably ran on the push already (e.g, we "promote" an existing build). So fyi this target_tasks method probably won't end up being used there.

Reply:
The target tasks method should be used for the push though right?  I renamed the key to `full` and added a comment on how I think it should work.  Hopefully that's enough good for now.